docs/hugo/content/reference/breaking-changes.md

```diff
@@ -0,0 +1,434 @@
+| DeliveryPolicy.Rules.Conditions.DeliveryRuleCookies                 | Renamed to Cookies (discriminator)                    | Update your YAML |
+| DeliveryPolicy.Rules.Conditions.DeliveryRuleHostName                | Renamed to HostName (discriminator)                   | Update your YAML |
+| DeliveryPolicy.Rules.Conditions.DeliveryRuleHttpVersion             | Renamed to HttpVersion (discriminator)                | Update your YAML |
+| DeliveryPolicy.Rules.Conditions.DeliveryRuleIsDevice                | Renamed to IsDevice (discriminator)                   | Update your YAML |
+| DeliveryPolicy.Rules.Conditions.DeliveryRulePostArgs                | Renamed to PostArgs (discriminator)                   | Update your YAML |
+| DeliveryPolicy.Rules.Conditions.DeliveryRuleQueryString             | Renamed to QueryString (discriminator)                | Update your YAML |
+| DeliveryPolicy.Rules.Conditions.DeliveryRuleRemoteAddress           | Renamed to RemoteAddress (discriminator)              | Update your YAML |
+| DeliveryPolicy.Rules.Conditions.DeliveryRuleRequestBody             | Renamed to RequestBody (discriminator)                | Update your YAML |
+| DeliveryPolicy.Rules.Conditions.DeliveryRuleRequestHeader           | Renamed to RequestHeader (discriminator)              | Update your YAML |
+| DeliveryPolicy.Rules.Conditions.DeliveryRuleRequestMethod           | Renamed to RequestMethod (discriminator)              | Update your YAML |
+| DeliveryPolicy.Rules.Conditions.DeliveryRuleRequestScheme           | Renamed to RequestScheme (discriminator)              | Update your YAML |
+| DeliveryPolicy.Rules.Conditions.DeliveryRuleRequestUri              | Renamed to RequestUri (discriminator)                 | Update your YAML |
+| DeliveryPolicy.Rules.Conditions.DeliveryRuleServerPort              | Renamed to ServerPort (discriminator)                 | Update your YAML |
+| DeliveryPolicy.Rules.Conditions.DeliveryRuleSslProtocol             | Renamed to SslProtocol (discriminator)                | Update your YAML |
+| DeliveryPolicy.Rules.Conditions.DeliveryRuleUrlFileExtension        | Renamed to UrlFileExtension (discriminator)           | Update your YAML |
+| DeliveryPolicy.Rules.Conditions.DeliveryRuleUrlFileName             | Renamed to UrlFileName (discriminator)                | Update your YAML |
+| DeliveryPolicy.Rules.Conditions.DeliveryRuleUrlPath                 | Renamed to UrlPath (discriminator)                    | Update your YAML |
+
+
+## Compute v2020-12-01
+
+### VirtualMachineScaleSet
+
+| Property                                                 | Change (Reason) | Impact |
+| -------------------------------------------------------- | --------------- | ------ |
+| VirtualMachineProfile.NetworkProfile.Id                  | Removed (??)    | ??     |
+| VirtualMachineProfile.NetworkProfile.IpConfigurations.Id | Removed (??)    | ??     |
```

Comment:
Both of these `Id` fields are now `Reference` fields which allow for linking to a resource in K8s instead of only in Azure. This was fixed because of better detection of IDs from Swagger.

Reply:
Updated.